modules/client/test/unit/client/utils.test.ts

```diff
@@ -117,6 +117,24 @@ describe("Test client utils", () => {
           .NON_ZERO_GRANT_UPGRADE_PLUGIN_PERMISSION_CALL_VALUE,
       ]);
     });
+    it("should return an error to of the action is not the DAO", async () => {
```

Comment:
```suggestion
    it("should return an error if `to` is not the DAO address", async () => {
```
Fix other occurrences as well